packages/generator/tests/utilities/measurements/non_streaming_data_measurement/__init__.py

```diff
@@ -52,6 +62,10 @@
     "IO Array In", nims.DataType.IOResourceArray1D, ["resource1", "resource2"]
 )
 @measurement_service.configuration("Integer In", nims.DataType.Int32, 10)
+@measurement_service.configuration("Enum In", nims.DataType.Enum, Color.BLUE, enum_type=Color)
+@measurement_service.configuration(
+    "Enum Array In", nims.DataType.EnumArray1D, [1, 2], enum_type=Color
+)
```

Comment:
It might be an over-ask but because we showcase "protobuf enums" in our sample measurement, should we also test the Protobuf Enum type with our client to ensure we're not missing anything on that part?